superstruct/src/types.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  FieldValues,
+  ResolverResult,
+  UnpackNestedValue,
+} from 'react-hook-form';
+import { validate, Struct } from 'superstruct';
+
+type Options = Parameters<typeof validate>[2];
```

Comment:
oh, I see why you need to leave those type here, that's fine as well. probably need to include that in the doc, so user don't get confused where the type is, perhaps we should remove that type from the core repo.